pom.xml

```diff
@@ -129,6 +129,12 @@
       <version>3.2</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.googlecode.json-simple</groupId>
+      <artifactId>json-simple</artifactId>
+      <version>1.1.1</version>
+    </dependency>
```

Comment:
We already have the code in place that is parsing the JSON and maps it to Java model based on `jackson`.
I do not understand why we should introduce yet another JSON parsing library.

Reply:
ok, I will change this.